participants/HarborCloud.md

```diff
@@ -0,0 +1,56 @@
+
+## (0) Project Name and Team Slack Handles
+
+-HarborCloud
+-shihuajin111@gmail.com
+
+## (1) Brief Project Description
+
+The HarborCloud Platform is a decentralized file storage system. Use Lotus for hot storage. Convenient for quick retrieval, Filecoin is used as cold storage for archiving
+
+## (2) Link to Project UI
+
+http://harborcloud.xingjigangwan.com/
+
+## (3) What does your application/UI do?
+
+In the HarborCloud system. Users can register their own accounts. Store whatever data you want to store. Important data can be stored encrypted. Retrieve relevant data. Infrequently used data can be archived
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+lotus
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+N/A
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+100T
+
+## (7) How are you structuring the data?
+
+lotus.import data use aes encryption
```

Comment:
Lotus may be best suited for cold storage.  You may want to use Powergate or IPFS for the hot storage layer.